vars/create_windows_template_vars.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# The variables in this file are copied from file "test.yml" for creating Windows
+# OVF template testing. It's used with "windows/create_template_playbook_list.yml".
+# The commented out variables are optional.
+#
+exit_testing_when_fail: True
+
+# Set 'proxy:port' when downloading files from external servers requiring proxy on
+# your local machine.
+# http_proxy_localhost: ""
+
+# vCenter Server information is optional
+# vcenter_hostname: "xx.xx.xx.xx"
+# vcenter_username: "xxxxxx@xx.xx"
+# vcenter_password: "xxxxxx"
+# datacenter: "xxxxxx"
+
+# ESXi Server information is mandatory
+esxi_hostname: "xx.xx.xx.xx"
+esxi_username: "root"
+esxi_password: "xxxxxx"
+
+datastore: "datastore1"
+# validate_certs: False
+
+vm_name: "windows_template_ansible"
+vm_username: "test"
+vm_password: "B1gd3m0z"
+# vm_folder: ''
+new_vm: True
+hardware_version: 16
+
+# Guest ID:
+# Windows 10 32bit: Windows9Guest,
+# Windows 10 64bit: Windows9_64Guest,
+# Windows 11 64bit: Windows11_64Guest,
```

Comment:
This guest id hasn't been published yet. So I suggest removing it for now. And windows guest ids' first letter should be lower case.

Reply:
Ok, that's right. While upper case ID also work.